manager_dashboard/manager_dashboard/usergroup.html

```diff
@@ -0,0 +1,201 @@
+        <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#add-usergroup-modal">
+          Add New User Group
+        </button>
+      </div>
+
+      <div class="container" >
+        <!-- manage user group -->
+        <div class="row justify-content-center" id="user-group-management" style="display: block">
+          <div id="groups" class="col-md-12 user-group--data">
+            <h2>User Groups</h2>
+            <p>These groups are currently set up in the MapSwipe app.</p>
+            <table id="userGroupsTable" class="table table-striped table-bordered">
+              <thead>
+                <tr class="thead-inverse">
+                  <th>Name</th>
+                  <th>Description</th>
+                  <th>Members</th>
+                </tr>
+              </thead>
+              <tbody>
+                <!-- will be added by function -->
+              </tbody>
+            </table>
+            <!-- Group Members -->
+            <h2>User Group Members</h2>
+            <p>Click the button for one of the user group above to display team members. (Note: this shows all contributions and thus might also contain contributions to non-private projects.</p>
+            <table id="groupMembersTable" class="table table-striped table-bordered">
+              <thead>
+                <tr class="thead-inverse">
+                  <th>User ID</th>
+                  <th>User Name</th>
+                  <th>Project Contributions</th>
+                  <th>Group Contributions</th>
+                  <th>Task Contributions</th>
+                </tr>
+              </thead>
+              <tbody>
+                <!-- will be added by function -->
+              </tbody>
+            </table>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <footer class="footer">
+      <div class="container" align="center">
+        <span class="text-muted">Copyright © 2019 MapSwipe</span>
+      </div>
+    </footer>
+
+    <div class="modal fade" tabindex="-1" role="dialog" id="add-usergroup-modal" aria-labelledby="Add user group" aria-hidden="true">
+      <div class="modal-dialog modal-lg" role="document">
+        <div class="modal-content">
+          <div class="modal-header">
+            <h5 class="modal-title">
+              Create a new user group
+            </h5>
+            <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+              <span aria-hidden="true">&times;</span>
+            </button>
+          </div>
+          <div class="modal-body">
+            <div class="row">
+              <div class="col-md-12">
+                <h3>User group Information</h3>
+                <form class="form-style-7" id="usergroup-form">
+                  <ul>
+                    <li>
+                      <label for="usergroup-name">Name</label>
+                      <input type="text" name="usergroupName" id="usergroup-name" maxlength="30" required>
```

Comment:
Should this value be cleaned/validated before being accepted? What would happen if a user input something like `../broken.value/` or `group/child`?

Reply:
Will add sanitization